src/studio/src/designer/frontend/shared/version-control/cloneModal.tsx

```diff
@@ -60,6 +65,9 @@ function CloneModal(props: ICloneModalProps) {
 
   React.useEffect(() => {
     checkIfDataModelExists();
```

Comment:
Some further cleanup if you feel like it;

Could move the `checkIfDataModelExists` function into the `useEffect`, since its only used within the useEffect thats the recommended approach (although I cannot seem to find the docs where this is stated right now)

Reply:
https://reactjs.org/docs/hooks-faq.html#is-it-safe-to-omit-functions-from-the-list-of-dependencies